apps/passport-server/src/util/devconnectTicket.ts

```diff
@@ -8,11 +8,11 @@ export function pretixTicketsDifferent(
   oldTicket: DevconnectPretixTicket,
   newTicket: DevconnectPretixTicket
 ): boolean {
-  if (oldTicket.is_deleted !== newTicket.is_deleted) {
+  if (oldTicket.full_name !== newTicket.full_name) {
     return true;
   }
 
-  if (oldTicket.full_name !== newTicket.full_name) {
+  if (oldTicket.is_deleted !== newTicket.is_deleted) {
     return true;
```

Comment:
Is there something significant about the change in order of these checks?

Reply:
not really, I was just matching some order I was seeing somewhere else